tonic-build/src/server.rs

```diff
@@ -492,7 +475,7 @@ fn generate_unary<T: Method>(
         let max_encoding_message_size = self.max_encoding_message_size;
         let inner = self.inner.clone();
         let fut = async move {
-            let inner = inner.0;
+            let inner = inner;
```

Comment:
What is the purpose of this and the corresponding sentences?

Reply:
Good catch, fixed!